src/routes/Review/index.jsx

```diff
@@ -146,10 +150,10 @@ const Review = ({
       "form.basicInfo.selectedDevice.option.length",
       1
     );
-    const additionalPaymentInfo =
-      workType?.selectedWorkType === "Website Design"
-        ? `\n${numOfPages} Pages\n${numOfDevices} Devices`
-        : "";
+    const additionalPaymentInfo = "";
+    // workType?.selectedWorkType === "Website Design"
```

Comment:
can we delete this comment out code or add a TODO: to explicitly describe when we can uncomment it?

Reply:
Resolved